sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/LibraryTasks.scala

```diff
@@ -78,6 +79,7 @@ object LibraryTasks {
         webpackResourceFiles ++ compileResources
       val cacheLocation = streams.value.cacheDirectory / s"${stage.key.label}-webpack-libraries"
       val extraArgs = (webpackExtraArgs in stage).value
+      val webpackMode = if (stage.key == fastOptJS.key) Webpack.DevelopmentMode else Webpack.ProductionMode
```

Comment:
Consider reading `(scalaJSLinkerConfig in stage).value.semantics.productionMode` instead.

Reply:
Thanks @sjrd. Will do